### PR TITLE
Using empty strings instead of nil 

### PR DESCRIPTION
**Pull request by igorescobar**
Addressing issue #11

### CODE REVIEW

File: lib/config_cat/user.ex [hunk 1]
```diff
@@ -1,6 +1,6 @@
 defmodule ConfigCat.User do
   @enforce_keys :identifier
-  defstruct [:identifier, email: nil, country: nil, custom: nil]
+  defstruct [:identifier, country: "", email: "", custom: ""]
```
Comment (randycoulman) [inline lib/config_cat/user.ex]: Should custom be `%{}` instead of an empty string? As I understand it, custom is a nested map of additional properties.

**Review by randycoulman — CHANGES_REQUESTED** [review]
This looks great! The only thing is that I think the default value for `custom` should be an empty map (`%{}`), not an empty string.  That'll affect several places in the updated code, I think.

File: lib/config_cat/user.ex (hunk 1 above)
Comment (igorescobar) [reply lib/config_cat/user.ex]: Well spotted @randycoulman , fixed 👍

**Review by randycoulman — APPROVED** [review]
Changes LGTM!

File: lib/config_cat/user.ex (hunk 1 above)
Comment (igorescobar) [reply lib/config_cat/user.ex]: Wait... found more stuff

**Pull request merged**